paseos/tests/communication_window_test.py

```diff
@@ -52,17 +56,61 @@ def test_communication_link_sat_to_ground():
 
     # Add communication link
     ActorBuilder.add_comm_device(sentinel2B, device_name="link1", bandwidth_in_kbps=1)
-    transmitted = 0
+    return sentinel2B, maspalomas_groundstation
+
+
+def test_find_next_window():
+
+    # Test window from other test is found
+    t0 = pk.epoch_from_string("2022-Oct-27 21:00:00")
+    sentinel2B, maspalomas_groundstation = setup_sentinel_example(t0)
+
+    start, length, transmittable_data = find_next_window(
+        sentinel2B,
+        link_name="link1",
+        target_actor=maspalomas_groundstation,
+        search_window_in_s=600,
+        t0=t0,
+    )
+
+    assert np.isclose(start.mjd2000, 8335.87835648148)
+    assert np.isclose(length, 731.9999999657739, rtol=0.01, atol=3.0)
```

Comment:
also check that the transmittable data matches with the assigned bandwidth and length?